### PR TITLE
Fix oci8_forge

### PR DESCRIPTION
**Pull request by kenjis**
* Oralce does not have `create table if exists`
* Oracle has only NUMBER for integer

### CODE REVIEW

**Comment by narfbg** [comment]
Any changes to the oci8 driver should be applied to pdo_oci too.

Also, while there is no INTEGER type per se, Oracle does recognize the keyword and automatically creates NUMBER fields instead - we don't do aliases for aliases.

... and this is why you shouldn't submit patches that aggregate multiple changes.

**Comment by kenjis** [comment]
> Also, while there is no INTEGER type per se, Oracle does recognize the keyword and automatically creates NUMBER fields instead - we don't do aliases for aliases.

When I run migration, if I remove the `_attr_type` function int this PR, I got the following error:
```
Database error: A Database Error Occurred

	Error Number: 907
	ORA-00907: 右カッコがありません。
	CREATE TABLE "migrations" (
	"version" BIGINT(20) NOT NULL
)
	Filename: libraries/Migration.php
	Line Number: 173
```
But the code with this PR, I can run migration.

**Comment by narfbg** [comment]
Fine.

Please make the same changes for pdo/oci.

**Comment by kenjis** [comment]
I pushed.

File: system/database/drivers/pdo/subdrivers/pdo_oci_forge.php [hunk 1]
```diff
@@ -54,6 +54,13 @@ class CI_DB_pdo_oci_forge extends CI_DB_pdo_forge {
 	protected $_create_database	= FALSE;
 
 	/**
+	 * CREATE TABLE IF statement
+	 *
+	 * @var	string
+	 */
+	protected $_create_table_if	= FALSE;
```
Comment (narfbg) [inline system/database/drivers/pdo/subdrivers/pdo_oci_forge.php]: Look 15 lines below ... this property already exists.

**Comment by kenjis** [comment]
I'm sorry. I fixed it.

**Pull request merged**